index.html

```diff
@@ -4,9 +4,188 @@
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300&display=swap" rel="stylesheet">
+    <!-- CSS file -->
+    <link rel="stylesheet" href="styles/style.css">
+    <!-- JS file -->
+    <script src="scripts/main.js"></script>
 </head>
-<body>
-    
+<body id="body">
+    <main>
+    <div class="sidebar"id="hide-menu">
+        <a class="close-sidebar" id="close-nav"><i class="fa-solid fa-xmark"></i></a>
+        <ul >
+            <a href="index.html">
+                <div class="nav-selector">
+                    <li>Home</li>
+                    <div class="guide_bar"></div>
+                </div>
+            </a>
+            <a href="about.html">
+                <div class="nav-selector">
+                    <li>About</li>
+                    <div class="guide_bar"></div>
+                </div>
+            </a>
+            <a href="#program">
+                <div class="nav-selector">
+                    <li>Main Program</li>
+                    <div class="guide_bar"></div>
+                </div>
+            </a>
+            <a href="#speakers">
+                <div class="nav-selector">
+                    <li>Speakers</li>
+                    <div class="guide_bar"></div>
+                </div>
+            </a>
+            <a href="#partners">
+                <div class="nav-selector">
+                    <li>Partners</li>
+                    <div class="guide_bar"></div>
+                </div>
+            </a>
+        </ul>
+    </div>
+
+    <section class="land-page">
+            <div>
+                <nav class="top-menu">
+                    <ul>
+                        <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+                        <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                        <li><a href="#">Spanish</a></li>
+                        <li><a href="#">My Page</a></li>
+                        <li><a href="#">Logout</a></li>
+                    </ul>    
+                </nav>
+            </div>
+            <div class="header-menu">
+                <div class="probando">
+                    <div>
+                        <a href="#"><img class="logo-desktop" src="images/new_logo.png" width="120" height="120" alt="main-logo" href="#"></a>
+                    </div>
+                    <nav>
+                        <ul class="menus">
+                            <li><a href="about.html">About</a></li>
+                            <li><a href="#">Program</a></li>
+                            <li><a href="#">Join</a></li>
+                            <li><a href="#">Sponsor</a></li>
+                            <li><a href="#">News</a></li>
+                            <li class="special-li"><a href="#">CC Campaign</a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
```

Comment:
- [ ] These two bars are the navbar and they would be better grouped in the same `<nav>` tag together, and maybe they should not be part of this header section. There is no need to have two navbars. Please group the navbars together in the same `<nav>` tag.